themes/aframe/source/css/home.styl

```diff
@@ -17,9 +17,6 @@ ul
 .home
   display flex
   flex-wrap wrap
-  padding 30px
```

Comment:
I would keep the 30px padding (or slightly smaller), but we can remove the 8% piece.